yb-voyager/cmd/constants.go

```diff
@@ -183,8 +184,10 @@ const (
 	MIGRATION_CAVEATS_CATEGORY_DESCRIPTION            = "Migration Caveats highlights the current limitations with the migration workflow."
 	UNSUPPORTED_QUERY_CONSTRUCTS_CATEGORY_DESCRIPTION = "Source database queries not supported in YugabyteDB, identified by scanning system tables."
 	UNSUPPPORTED_PLPGSQL_OBJECT_CATEGORY_DESCRIPTION  = "Source schema objects having unsupported statements on the target YugabyteDB in PL/pgSQL code block"
-	SCHEMA_SUMMARY_DESCRIPTION                        = "Objects that will be created on the target YugabyteDB."
-	SCHEMA_SUMMARY_DESCRIPTION_ORACLE                 = SCHEMA_SUMMARY_DESCRIPTION + " Some of the index and sequence names might be different from those in the source database."
+	PERFORMANCE_OPTIMIZATIONS_CATEGORY_DESCRIPTION    = "Features of the source database that may not perform optimally on the target YugabyteDB."
```

Comment:
How about `Changes to schema/queries to optimize performance on YugabyteDB. `

Reply:
`Recommendations to schema/queries to optimize performance on YugabyteDB.` ?